### PR TITLE
Modular veil renders

### PR DESCRIPTION
**Pull request by Jordie0608**
Veil renders that spawn atoms via vars instead of hardcoded paths.

### CODE REVIEW

File: code/game/gamemodes/wizard/artefact.dm [hunk 1]
```diff
@@ -12,8 +12,18 @@
...
 	hitsound = 'sound/weapons/bladeslice.ogg'
+	var/charged = 1
+	var/spawn_type = /obj/singularity/narsie/wizard
+	var/spawn_amt = 1
+
+/obj/item/weapon/veilrender/attack_self(mob/user as mob)
+	if(charged)
+		new /obj/effect/rend(get_turf(usr), spawn_type, spawn_amt)
+		charged = 0
+		user.visible_message("<span class='userdanger'>[src] hums with power as [usr] deals a blow to reality itself!</span>")
```
Comment (Anonus) [inline code/game/gamemodes/wizard/artefact.dm]: The "to reality itself" part should change based on a var, as the veal render used "to hunger itself" there.

**Comment by Anonus** [comment]
The rend's desc should be a var so you can make it custom for various renders.

**Comment by RemieRichards** [comment]
Descriptions are already a variable...

**Comment by AlexanderUlanH** [comment]
I think he means the text that's displayed when the render's used.  That's hardcoded into the proc.

**Comment by Anonus** [comment]
The second comment refers to the desc of the spawned rend object, to be clear.

**Comment by TheVekter** [comment]
Does this mean we can make veal renders that spit out stuff other than cows?

**Comment by Jordie0608** [comment]
Yes.

**Comment by TheVekter** [comment]
Clown rifts. Finally.

**Comment by scrdest** [comment]
Clown render, you say? 

!http://i.imgur.com/P84TFz9.png?1

**Pull request merged**